packages/pcd/eddsa-frog-pcd/test/EdDSAFrogPCD.spec.ts

```diff
@@ -76,6 +76,38 @@ describe("EdDSA frog should work", function () {
     expect(deserialized).to.deep.eq(frog);
   });
 
+  it("should be able to compatibly deserialize a saved PCD", async function () {
+    const savedPCD =
```

Comment:
worth adding a comment describing how you got this saved pcd. e.g. something like

```
// generated on feb 9 2024 using pcd package <package> at commit <commit hash>
```

Reply:
Good idea.  I'll base it on the parent commit from main, since the commits on my branch are going to get squashed.